pkg/webhook/patch.go

```diff
@@ -360,15 +379,31 @@ func addPrometheusConfigMap(pod *corev1.Pod, app *v1beta2.SparkApplication) []pa
 	name := config.GetPrometheusConfigMapName(app)
 	volumeName := name + "-vol"
 	mountPath := config.PrometheusConfigMapMountPath
+	port := config.DefaultPrometheusJavaAgentPort
```

Comment:
Can we rename the method from `addPrometheusConfigMap` to `getPrometheusConfigPatches` given it's doing more than just adding the ConfigMap?

Reply:
Agress. Done.